### PR TITLE
WRQ-15997: Fix a rule: no-new-object(deprecated) -> no-object-constructor

### PR DESCRIPTION
**Pull request by hong6316**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] I have run automated CLI testing and it is passed
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
In WRQ-15997, an deprecated rule has been found.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Fix a rule: no-new-object(deprecated) -> no-object-constructor

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-15997

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

**Comment by juwonjeong** [comment]
Please resolve the conflicts

**Review by juwonjeong — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 The following is a curated list of changes in the Enact eslint config:
 
+## [unreleased]
+
+* Repleced deprecated rule `no-new-object` with `no-object-constructor`.
```
Comment (MikyungKim) [inline CHANGELOG.md:7]:
```suggestion
* Replaced deprecated rule `no-new-object` with `no-object-constructor`.
```

Comment (juwonjeong) [reply CHANGELOG.md]: It's fixed.

**Pull request merged**